### PR TITLE
Improved an example on webRequest.StreamFilter.ondata page

### PR DESCRIPTION
**Pull request by def00111**
#### Summary

Improved an example on webRequest.StreamFilter.ondata page
#### Motivation

For better performance (only 1 for...of loop needed instead of 2)
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Para505** [comment]
j

Stephen Balogh

On Feb 27, 2022, at 8:10 PM, github-actions[bot] 

﻿
Preview URLs

https://pr13297.content.dev.mdn.mozit.cloud/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/StreamFilter/ondata
Flaws

None! 🎉

External URLs

URL: /en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/StreamFilter/ondata
Title: webRequest.StreamFilter.ondata
on GitHub

No new external URLs

File: files/en-us/mozilla/add-ons/webextensions/api/webrequest/streamfilter/ondata/index.md [hunk 1]
```diff
@@ -234,21 +234,19 @@ This example combines all buffers into a single buffer:
 ```js
 function listener(details) {
   let filter = browser.webRequest.filterResponseData(details.requestId);
-  let decoder = new TextDecoder("utf-8");
+  let decoder = new TextDecoder(); // defaults to utf-8
```
Comment (rpl) [inline files/en-us/mozilla/add-ons/webextensions/api/webrequest/streamfilter/ondata/index.md:237]: nit, personally I would change this back to a string parameter, in the end the comment is (ironically) longer than the string parameter that is is replacing ;-)

**Review by rpl — COMMENTED** [review]
@rebloor @def00111 my apologies I completely missed this review request and just noticed it now while I was going through github notifications.

This looks ok, just a couple of nits from my perspective.

**Comment by sideshowbarker** [comment]
@def00111 This PR is waiting on your responses to review comments

**Comment by rebloor** [comment]
@def00111 it appears that the code example has been updated since you submitted these changes. As you haven't had the opportunity to look at the feedback since April, I've closed this PR. If you feel the code example could be improved, please feel free to submit a new change. Thank you.

**Pull request closed without merging**